sktime/utils/validation/series.py

```diff
@@ -153,3 +158,45 @@ def check_equal_time_index(*ys):
 
         if not first_index.equals(y.index):
             raise ValueError("Some (time) indices are not the same.")
+
+
+def _is_int_index(index):
+    """Check if index type is one of pd.RangeIndex or pd.Int64Index"""
+    return type(index) in VALID_INDEX_TYPES[:2]
```

Comment:
Let's make this more explicit here, because the order of `VALID_INDEX_TYPES` may change